apps/src/code-studio/progressRedux.js

```diff
@@ -317,7 +317,8 @@ export function processedStages(stages, isPlc) {
  * based on the server's response data.
  */
 const userProgressFromServer = (state, dispatch, userId = null) => {
```

Comment:
`state` is now the entire state object and not just the progress portion, this was needed to access `viewAs` property